modular_skyrat/modules/primitive_fun/code/big_mortar.dm

```diff
@@ -0,0 +1,116 @@
+/obj/structure/large_mortar
+	name = "large mortar"
+	desc = "A large bowl perfect for grinding or juicing a large number of things at once."
+	icon = 'modular_skyrat/modules/primitive_fun/icons/cooking_structures.dmi'
+	icon_state = "big_mortar"
+	density = TRUE
+	anchored = TRUE
+	max_integrity = 100
+	pass_flags = PASSTABLE
+	custom_materials = list(/datum/material/wood = MINERAL_MATERIAL_AMOUNT * 10)
+	/// The maximum number of items this structure can store
+	var/maximum_contained_items = 5
+
+/obj/structure/large_mortar/Initialize(mapload)
+	. = ..()
+	create_reagents(200, OPENCONTAINER)
+
+/obj/structure/large_mortar/examine(mob/user)
+	. = ..()
+	. += span_notice("It currently contains <b>[LAZYLEN(contents)]/[maximum_contained_items] items.")
+	. += span_notice("It can be (un)secured with <b>Right Click</b>")
+	. += span_notice("You can empty all of the items out of it with <b>Alt Click</b>")
+
+/obj/structure/large_mortar/Destroy()
+	drop_everything_contained()
+	return ..()
+
+/obj/structure/large_mortar/AltClick(mob/user)
+	if(!LAZYLEN(contents))
+		balloon_alert(user, "nothing inside")
+		return
+	drop_everything_contained()
+	balloon_alert(user, "removed all items")
+
+/// Drops all contents at the mortar
+/obj/structure/large_mortar/proc/drop_everything_contained()
+	if(!LAZYLEN(contents))
+		return
+	for(var/obj/target_item in contents)
+		target_item.forceMove(get_turf(src))
+		LAZYREMOVE(contents, target_item)
```

Comment:
when you remove something from contents, it should automatically do this-- you don't need LAZYREMOVE here.